packages/agoric-cli/src/lib/bundles.js

```diff
@@ -11,7 +11,7 @@ import { ZipReader } from '@endo/zip';
 /** @import {Bundle} from '@agoric/swingset-vat'; */
 /** @import {CoreEvalPlan} from '@agoric/deploy-script-support/src/writeCoreEvalParts.js' */
 
-const PACKAGE_NAME_RE = /(?<packageName>.*-v[\d.]+)\//;
+const PACKAGE_NAME_RE = /(?<packageName>.*-v[\d.]+(?:-n\d+)?)\//;
```

Comment:
seems pretty low level to JS ecosystem. is there someplace from which we could import it?

Reply:
These names are generated by the compartment mapper and can technically be anything. I’m going to land this since it’s the smallest expedient, but it might be better to just remove the check so we don’t accidentally break if we need to change the scheme.